### ISSUE TITLE
rename project now that non-collection stuff is allowed

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-library-compat#317]
see discussion on https://github.com/scala/scala-collection-compat/issues/219

name suggestions so far: `scala-2.13-compat`, `scala-stdlib-compat`, `scala-compat`

**Comment by SethTisue** [comment]
`scala-crossbuilding`, maybe

**Comment by martijnhoekstra** [comment]
my bikeshed would be `scala-crosscompat` or mauve.

**Comment by ekrich** [comment]
I like `scala-stdlib-compat` FWIW.

**Comment by lrytz** [comment]
`scala-library-[compat|crossbuilding|crosscompat]` as it's for `scala-library`?

**Comment by NthPortal** [comment]
I'm for `scala-library-compat` or `scala-library-crosscompat`

**Comment by lrytz** [comment]
Seeing PRs trickle in with backports from the 2.13 library (thanks, @NthPortal), I find it a little ironic that the 2.11 / 2.12 standard libraries can acquire new classes (through this module), while the one in 2.13 (and 3.0) can't.

Should we consider extending the scope of this module to `scala-library-future` where we can also add new classes for 2.13 / 3.0?

**Comment by julienrf** [comment]
> Should we consider extending the scope of this module to `scala-library-future` where we can also add new classes for 2.13 / 3.0?

Given that there will be a lot of time before we can get a 2.14/3.1, yes, that would probably be useful.

**Comment by SethTisue** [comment]
I think `scala-library-compat` is good

> Should we consider extending the scope of this module

Sure, I don't see why not.

**Comment by lrytz** [comment]
> Sure, I don't see why not.

We'd have to make it clear that everything added to the 2.13 release here would eventually end up in the standard library, so we'd have to be cust as conservative.

**Comment by SethTisue** [comment]
I have renamed the repo and submitted a pair of PRs: #327 to do the renaming, and #328 to update the readme. once #327 has been merged, I can do the publishing.

**Comment by SethTisue** [comment]
> Should we consider extending the scope of this module to scala-library-future where we can also add new classes for 2.13 / 3.0?

I've made a separate ticket on this: https://github.com/scala/scala-library-compat/issues/329
